### DISCUSSION TITLE
how to best record monument or stickup on groundwater well installation

### DISCUSSION THREAD

**Discussion by hardreddata** [australiangeomechanics/agsformat#82] [Q&A]
We are wondering how to best record monument or stickup information for groundwater well installations.

As this is above the ground and environmental maybe I am out of scope?

We would like it to visually show on the log which I guess implies some kind of lookup or switch and not just a remark. 

Advice welcome.

Thanks.

**Comment by SolidRockProgrammer** [comment]
Hi
Water depth measurements in MOND should be from ground surface.
You could transmit the well construction in the PIPE group.
Also, could put a note in MONG_REM.
regards
Phil Wade
